lib/fastly_api.php

```diff
@@ -1651,7 +1651,7 @@ public function API_purge ( $url ) {
 	}
 
 	/*
-	 * POST /service/<id>/purge_all
```

Comment:
I don't think this change is correct. `POST` is the correct verb when attempting a `purge_all`